### PR TITLE
Add Seitron integration page

### PR DESCRIPTION
**Pull request by seitronspa**
## Proposed change

Added logos for Seitron new integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
This PR is targeting the wrong branch. As this depends on a new feature in a "next" release of Home Assistant, it should target the `next` branch.

Please change the target of this PR to the `next` branch and git rebase your changes onto the `next` branch.

../Frenck

File: source/_integrations/seitron.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ha_codeowners:
+  - '@stefano-vardanega'
+ha_config_flow: true
+ha_platforms:
+  - climate
+ha_integration_type: integration
+---
+
+The `Seitron` integration allows Home Assistant to connect to [Seitron](https://www.seitron.com) thermostats, reporting and setting its status.
+The integration implements the following platforms:
```
Comment (frenck) [inline source/_integrations/seitron.markdown]:
```suggestion
The Seitron integration allows Home Assistant to connect to [Seitron](https://www.seitron.com) thermostats, to report and set their status.
The integration implements the following platforms:
```

File: source/_integrations/seitron.markdown [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+- Climate
+
+{% include integrations/config_flow.md %}
+
+
+## Climate
+
+The `Seitron` climate platform provides current and target temperature information for the heating system, boiler status and HVAC mode.
```
Comment (frenck) [inline source/_integrations/seitron.markdown]:
```suggestion
The Seitron climate platform provides current and target temperature information for the heating system, boiler status, and HVAC mode.
```

File: source/_integrations/seitron.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+### Integration services
+
+This integration supports the following services (see [Climate](/integrations/climate/)).
+
+- [`set_temperature`](/integrations/climate/#service-climateset_temperature)
+- [`set_hvac_mode`](/integrations/climate/#service-climateset_hvac_mode)
+  - `heat` for thermostat winter mode
+  - `cool` for thermostat summer mode
+  - `auto` for thermostat schedule mode (heat/cool)
+  - `off` for thermostat defrost mode
```
Comment (frenck) [inline source/_integrations/seitron.markdown:38]: That seems incorrect. Defrost, winter, summer modes sound like presets or other behavior and not HVAC modes?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/seitron.markdown (hunk 3 above)
Comment (seitronspa) [reply source/_integrations/seitron.markdown:38]: Hi, winter/summer is just informal way to call HEAT and COOL in Italy, these are HVAC modes. If it's more clear I can remove the wording. Our thermostats don't have a real OFF mode. The user can turn off the thermostat, the physical thermostat show 'OFF' on its LCD, but it always act as defrost protection. This is a common behaviour for our customers.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**